frontend/apps/filemanager/filemanagerhistory.lua

```diff
@@ -223,6 +219,8 @@ function FileManagerHistory:onShowHist(search_info)
         covers_fullscreen = true, -- hint for UIManager:_repaint()
         is_borderless = true,
         is_popout = false,
+        title = self.hist_menu_title,
+        title_bar_fm_style = true,
```

Comment:
May be add a comment (or mention that in the commit message when you'll merge it) saying why we want the same style
`-- be sure coverbrowser thumbnails get the exact same size`